src/vbk/pop_service_impl.cpp

```diff
@@ -149,75 +142,14 @@ PoPRewards PopServiceImpl::getPopRewards(const CBlockIndex& pindexPrev, const Co
         rewardValue >>= halvings;
         if ((rewardValue != 0) && (halvings < 64)) {
             CScript key = CScript(r.first.begin(), r.first.end());
-            btcRewards[key] = config.POP_REWARD_COEFFICIENT * rewardValue;
+            btcRewards[key] = rewardValue;
```

Comment:
Pls calculate POP_REWARD_COEFFICIENT as (100%-60%) * BASE_REWARD and multiply rewardValue by this coefficient.